tensorflow/lite/experimental/acceleration/configuration/configuration.proto

```diff
@@ -622,6 +624,14 @@ message CPUSettings {
   optional int32 num_threads = 1 [default = -1];
 }
 
+// Arm NN Delegate Settings
```

Comment:
Hi. I noticed a very tiny issue in your comment here that I'd like to bring to your attention. In this specific block of text, I noticed that there's a missing full stop at the end of a sentence. I understand that this is a minor issue, but adding the full stop would help improve the readability and clarity of the text. 
The Google style guide recommends including punctuation in comments <https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar>.
Would you be willing to add it, or do you have any thoughts on this matter?